Code/Core/Containers/Sort.h

```diff
@@ -30,37 +30,80 @@ class AscendingCompareDeref
     }
 };
 
-// ShellSort
```

Comment:
Wondering why `ShellSort` is removed here - presumably there are no current uses for it, but it could potentially find uses in the future? Or even be used in a unit test of the new quicksort implementation?

Reply:
I'd be ok with deleting it if we determine it should never be used. I'd also be ok with moving it into the tests to have a simple way of verifying the sort/sorts we chose to keep (though it may be of limitted use since the sorts we're using are unstable and may not produce the same results for lists with "equal" items)